setup.py

```diff
@@ -33,8 +33,8 @@
 ]
 
 install_requires = [
-    'boto3>=1.7,<1.8',
-    'botocore>=1.8.34',
+    'boto3==1.9.37',
```

Comment:
Necessary to pin these specific versions in install_requires? Likely to become a source of user pain not too far down the road.

Reply:
Will fix.